src/command/config/clear.rs

```diff
@@ -0,0 +1,10 @@
+use anyhow::Result;
+
+use crate::command::RoverOutput;
+use houston as config;
+
+pub fn run() -> Result<RoverOutput> {
+    config::clear()?;
+    tracing::info!("Successfully cleared all configuration.");
+    Ok(RoverOutput::None)
```

Comment:
i could imagine it would be potentially useful to print this message to screen, not just tracing

Reply:
(i realize that tracing prints this message depending on verbosity.. i wonder if renaming Output to... Payload? might make more sense)

Reply:
the intention here is for `RoverStdout` to be in charge of all machine readable data, and for all human readable data to be handled by `tracing::info` or `tracing::warn`. `tracing::info` will print to stderr by default, the only reason it's included here is so that we have some human readable data to go along with our machine readable data when they are both being read by humans (this info log is not printed if stdout is redirected to a file).